specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/symsSync.json

```diff
@@ -0,0 +1,326 @@
+          },
+          {
+            "in": "body",
+            "name": "deleteBody",
+            "required": true,
+            "description": "The delete body",
+            "schema": {
+              "$ref": "#/definitions/metadataDeleteObject"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Metadata Request Accepted Response.",
+            "schema": {
+              "$ref": "#/definitions/metadataRequestAcceptedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/synapsemetastore/getrequeststatus": {
+      "get": {
+        "tags": [
+          "synapsemetastore"
+        ],
+        "operationId": "SynapseMetastore_GetRequestStatus",
+        "description": "Gets status of a request",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/SynapseMetastore_GetRequestStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request Success Response.",
+            "schema": {
+              "$ref": "#/definitions/successResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorContract": {
+      "title": "Error details.",
+      "description": "Contains details when the response code indicates an error.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The error details.",
+          "$ref": "#/definitions/ErrorResponse"
+        }
+      }
+    },
+    "ErrorResponse": {
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorDetail"
+          }
+        }
+      }
+    },
+    "ErrorDetail": {
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        }
+      }
+    },
+    "metadataRequestAcceptedResponse": {
```

Comment:
>metadataRequestAcceptedResponse [](http://example.com/codeflow?start=5&length=31)

We use the word "Metastore" for client name "SynapseMetastore" but here we use the word "metadata". It looks inconsistent to me. This is by design?